src/components/dialogs/DialogAdvanced.vue

```diff
@@ -47,10 +47,20 @@ export default {
 			default: 'Advanced',
 		},
 	},
+	emits: ['close', 'input'],
 	data() {
 		return {}
 	},
-	computed: {},
+	computed: {
+		localValue: {
+			get() {
+				return this.value
+			},
+			set(value) {
+				this.$emit('input', value)
```

Comment:
In each component you should emit `update:modelValue` not `input` and use `modelValue` not `value` prop as of vue 3 specs. It's ok to use a computed with get/set by the way. Apply this changes to all the components that use `input` and `value`. Remember to also update `emits` according to this